multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2766,84 +2766,52 @@ def load(
 
         return predictor
 
-    def dump_timm_image(
-        self,
-        path: str,
-    ):
+    def dump_multimodal(self, path: Optional[str] = None):
```

Comment:
`dump_multimodal` may not be intuitive for some user cases, e.g., image only. How about calling it `dump_model`?

Reply:
Done

Reply:
The singular form sounds good.